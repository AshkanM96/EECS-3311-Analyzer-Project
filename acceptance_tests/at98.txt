difference
integer_constant (2)
integer_constant (5)
evaluate
type_check
reset
difference
boolean_constant (false)
integer_constant (5)
evaluate
type_check
reset
difference
integer_constant (2)
boolean_constant (true)
evaluate
type_check
reset
difference
open_set_enumeration
integer_constant (2)
close_set_enumeration
integer_constant (2)
evaluate
type_check
reset
difference
integer_constant (2)
open_set_enumeration
integer_constant (2)
close_set_enumeration
evaluate
type_check
reset
difference
open_set_enumeration
integer_constant (2)
close_set_enumeration
boolean_constant (true)
evaluate
type_check
reset
difference
boolean_constant (true)
evaluate
close_set_enumeration
open_set_enumeration
integer_constant (3)
close_set_enumeration
evaluate
type_check
reset
difference
open_set_enumeration
integer_constant (3)
close_set_enumeration
open_set_enumeration
integer_constant (2)
close_set_enumeration
evaluate
type_check
reset
open_set_enumeration
reset
difference
open_set_enumeration
integer_constant (2)
integer_constant (3)
close_set_enumeration
open_set_enumeration
integer_constant (2)
integer_constant (3)
close_set_enumeration
evaluate
type_check
reset
difference
open_set_enumeration
boolean_constant (true)
integer_constant (2)
close_set_enumeration
open_set_enumeration
boolean_constant (true)
integer_constant (2)
close_set_enumeration
evaluate
type_check
reset
