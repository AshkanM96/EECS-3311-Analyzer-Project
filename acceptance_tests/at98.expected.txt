  Expression currently specified: ?
  Report: Expression is initialized.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: (2 \ ?)
  Report: OK.
->integer_constant(5)
  Expression currently specified: (2 \ 5)
  Report: OK.
->evaluate
  Expression currently specified: (2 \ 5)
  Report: Error (Expression is not type-correct).
->type_check
  Expression currently specified: (2 \ 5)
  Report: (2 \ 5) is not type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (False \ ?)
  Report: OK.
->integer_constant(5)
  Expression currently specified: (False \ 5)
  Report: OK.
->evaluate
  Expression currently specified: (False \ 5)
  Report: Error (Expression is not type-correct).
->type_check
  Expression currently specified: (False \ 5)
  Report: (False \ 5) is not type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: (2 \ ?)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (2 \ True)
  Report: OK.
->evaluate
  Expression currently specified: (2 \ True)
  Report: Error (Expression is not type-correct).
->type_check
  Expression currently specified: (2 \ True)
  Report: (2 \ True) is not type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({2, ?} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({2} \ ?)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({2} \ 2)
  Report: OK.
->evaluate
  Expression currently specified: ({2} \ 2)
  Report: Error (Expression is not type-correct).
->type_check
  Expression currently specified: ({2} \ 2)
  Report: ({2} \ 2) is not type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: (2 \ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (2 \ {?})
  Report: OK.
->integer_constant(2)
  Expression currently specified: (2 \ {2, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (2 \ {2})
  Report: OK.
->evaluate
  Expression currently specified: (2 \ {2})
  Report: Error (Expression is not type-correct).
->type_check
  Expression currently specified: (2 \ {2})
  Report: (2 \ {2}) is not type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({2, ?} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({2} \ ?)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: ({2} \ True)
  Report: OK.
->evaluate
  Expression currently specified: ({2} \ True)
  Report: Error (Expression is not type-correct).
->type_check
  Expression currently specified: ({2} \ True)
  Report: ({2} \ True) is not type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (True \ ?)
  Report: OK.
->evaluate
  Expression currently specified: (True \ ?)
  Report: Error (Expression is not yet fully specified).
->close_set_enumeration
  Expression currently specified: (True \ ?)
  Report: Error (Set enumeration is not being specified).
->open_set_enumeration
  Expression currently specified: (True \ {?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: (True \ {3, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (True \ {3})
  Report: OK.
->evaluate
  Expression currently specified: (True \ {3})
  Report: Error (Expression is not type-correct).
->type_check
  Expression currently specified: (True \ {3})
  Report: (True \ {3}) is not type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({3, ?} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3} \ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({3} \ {?})
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({3} \ {2, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({3} \ {2})
  Report: OK.
->evaluate
  Expression currently specified: ({3} \ {2})
  Report: {3}
->type_check
  Expression currently specified: ({3} \ {2})
  Report: ({3} \ {2}) is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->open_set_enumeration
  Expression currently specified: {?}
  Report: OK.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({2, ?} \ nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({2, 3, ?} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({2, 3} \ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({2, 3} \ {?})
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({2, 3} \ {2, ?})
  Report: OK.
->integer_constant(3)
  Expression currently specified: ({2, 3} \ {2, 3, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({2, 3} \ {2, 3})
  Report: OK.
->evaluate
  Expression currently specified: ({2, 3} \ {2, 3})
  Report: {}
->type_check
  Expression currently specified: ({2, 3} \ {2, 3})
  Report: ({2, 3} \ {2, 3}) is type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
->difference
  Expression currently specified: (? \ nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({?} \ nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: ({True, ?} \ nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({True, 2, ?} \ nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({True, 2} \ ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: ({True, 2} \ {?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: ({True, 2} \ {True, ?})
  Report: OK.
->integer_constant(2)
  Expression currently specified: ({True, 2} \ {True, 2, ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: ({True, 2} \ {True, 2})
  Report: OK.
->evaluate
  Expression currently specified: ({True, 2} \ {True, 2})
  Report: Error (Expression is not type-correct).
->type_check
  Expression currently specified: ({True, 2} \ {True, 2})
  Report: ({True, 2} \ {True, 2}) is not type-correct.
->reset
  Expression currently specified: ?
  Report: OK.
